notebook/nbextensions.py

```diff
@@ -831,7 +849,43 @@ class DisableNBExtensionApp(ToggleNBExtensionApp):
     Usage
         jupyter nbextension disable [--system|--sys-prefix]
     """
-    _toggle_value = None
+    version = __version__
+    flags = {"py" : ({
+        "DisableNBExtensionApp" : {
+            "python" : True,
+        }}, "Disable an extension from a Python package name")
+    }
+    flags['python'] = flags['py']
+    aliases = {'section': 'ToggleNBExtensionApp.section'}
```

Comment:
I've added the `--user` and `--sys.prefix` flags. I don't think `--system` is necessary, since there's nothing for system config to override.